OpenMage/magento-lts
# Changed "unload" to "beforeunload" in extjs to avoid deprecation warning in developer console

### Description (*)
Updated the unload calls to be beforeunload in extjs (removes deprecation warning in chrome console).

### Manual testing scenarios (*)

1. Open console in chrome when in backend admin section.
2. Deprecated feature warning will be shown.
3. With this change the warning related to extjs no longer appears.


### Contribution checklist (*)
